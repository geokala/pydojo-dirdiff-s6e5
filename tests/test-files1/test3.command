./dirdiff tests/test-files1/test3.{a,b}
