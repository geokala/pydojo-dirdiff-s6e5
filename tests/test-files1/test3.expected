1 difference between directories:
test3.a contains c.txt that cannot be found in test3.b
